test/Microsoft.AspNetCore.Mvc.Core.Test/RequestFormLimitsAttributeTest.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Http.Features;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Mvc
+{
+    public class RequestFormLimitsAttributeTest
+    {
+        [Fact]
+        public void CreatesFormOptions_WithDefaults()
+        {
+            // Arrange
+            var formOptions = new FormOptions();
```

Comment:
Should we reflect over the properties to verify that we are in sync with properties exposed on `FormOptions`? So we don't miss adding it here if HttpAbstractions gets a new limit.